pkgs/applications/networking/owncloud-client/libre-graph-api-cpp-qt-client.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, mkDerivation
+, cmake
+, qtbase
+}:
+
+mkDerivation rec {
+  pname = "libre-graph-api-cpp-qt-client";
+  version = "0.13.2";
+
+  srcs = (fetchFromGitHub {
+    owner = "owncloud";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-gbrA8P+ukQAiF2czC2szw3fJv1qoPJyMQ72t7PqB5/s=";
+  });
```

Comment:
Remove parenthesis and this can also just use `src` instead of `srcs`.

Reply:
right, probably a refactoring relics when I wanted to build from two srcs, will refactor in 4.0.0

Reply:
done